modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/RelationshipObjectFieldBusinessType.java

```diff
@@ -93,14 +93,29 @@ public Object getValue(
 					NAME_OBJECT_RELATIONSHIP_ERC_OBJECT_FIELD_NAME,
 				objectField);
 
```

Comment:
I think this is not the correct way to fix the issue. 

The validation fails as I said before in the `DefaultObjectEntryManagerImpl.addObjectEntry` as it tries validating that mandatory fields through the `_objectEntryService.addObjectEntry(` statement found in the line 156. This issue is still happening (see the attached screenshot) 
![image](https://github.com/user-attachments/assets/8d9cdfa0-1c91-42c8-a74e-bfa5ddce0f4c)

Also, this PR doesn't fix the issue of the validation order. As stated by the LPD-36173 story description "...when the API request is sent, the creation of the object prioritizes the relationship fields in a specific order... should be nested Field -> Id -> ERC..." (the priority with the PR is nestedField -> ERC -> Id)

Please rework it.